solar/solar.py

```diff
@@ -113,9 +117,12 @@ def setup(self, static, state):
             dvars.append(cdfuse*Sfuse/Sw)
 
         constraints = [cda >= sum(dvars),
+                       self.propeller.T==T,
```

Comment:
Can you get rid of all the `self.` in the constraints list and make them separate variables like I do for `CL`?